### ISSUE TITLE
Crash on iOS with BT OBD2 Dongle

### ISSUE THREAD

**Issue by RalphS69** [EVNotify/EVNotify#240]
### New issue
- critical issue 
- follow-up on https://github.com/EVNotify/EVNotify/issues/215
- during initialization the app either crashes or runs into an initialization error

#### Hardware / Software:
- iPhone 6s with iOS 14.5.1
- OBD2 dongle: vLinker MC+ for iOS and Android
- Ioniq elctric classic
<details>
<summary>no successful connection</summary>

- The dongle is listed in the settings as **vLinker MC-IOS**
- when going back to the dashboard, the app
  - crashes, or
  - repeatedly says:
    - "initialization, please wait"
    - "OBDII error. Re-initializing, Please wait"
</details>

**Comment by GPlay97** [comment]
Is car fully turned on or charging?

Otherwise the dongle may not be supported, on iOS only one dongle is confirmed to work yet: https://github.com/EVNotify/EVNotify/wiki/Compatibility

**Comment by RalphS69** [comment]
- If the car is fully turned on: The app crashes
- If the car is off, the initialization fails repeatedly

I updated the initial comment accordingly.

Why is just one device supported? My dongle was not a cheap one, and works fine with car scanner.

**Comment by GPlay97** [comment]
Crash is weird, but I don't know if I have the chance to reproduce it locally. Will see in the upcoming days when I find time. On iOS, only one ble characterises can be used currently, technical limitation. This can only change in the upcoming v3 Version with native code

**Comment by RalphS69** [comment]
Yeah. Occasionally I see quite short the "initialization, please wait" - than this app is gone. Would be nice if you could improve this somehow - I need this app to feed abrp with the actual car data.
BTW: Is the app still available for iOS? I cannot find it in the store anymore ...

**Comment by GPlay97** [comment]
Currently not, was taken away a few days ago by Apple due to expired credit card. It's expected that it'll be available again until end of this week hopefully

**Comment by RalphS69** [comment]
Not from my side. I am just the user who would like to use the app ...

**Comment by oscjan** [comment]
Same problem for me on iPhone 13 running ios 16.1.1 and OBD2 dongle: vLinker MC+ for iOS and Android. Cars VW ID.4 GTX.

**Comment by m3gg3** [comment]
same problem here. ios 16.3.1 and a supported dongle https://www.amazon.de/gp/product/B071D8SYXN/ref=as_li_ss_tl?ie=UTF8&psc=1&linkCode=sl1&tag=gplay97-21&linkId=68ed6e726985a720808ee22317d51f8b&language=de_DE

app opens then initialize and crashes

volkswagen id.4 gtx (3.0.5) and the vcar dongle has the newest firmware on it

**Comment by achris2000** [comment]
Same here. VW ID.3 Pro Performance (2.4), same Vgate obd stick, iphone 14 Pro Max 16.3.1.

**Comment by RalphS69** [comment]
evNotify is not available in the AppStore anymore (at least in DE). Why?

The app could not be used directly (due to the crash), but I could use it as remote monitor, retrieving the data in the car with an Android phone.

So if just this issue is the reason of the removal: Please add it again, its useful even with the BT connection not working.
